src/algorithm/MinimumBoundingCircle.cpp

```diff
@@ -45,7 +45,7 @@ namespace algorithm { // geos.algorithm
 std::unique_ptr<Geometry>
 MinimumBoundingCircle::getCircle()
 {
-    //TODO: ensure the output circle contains the extermal points.
```

Comment:
"extremal" is correct here

Reply:
Good catch. This was suggested too: `extermal ==> external, extremal`